res/controllers/midi-components-0.0.js

```diff
@@ -181,9 +181,11 @@
     PlayButton.prototype = new Button({
         unshift: function () {
             this.inKey = 'play';
+            this.onlyOnPress = true;
         },
         shift: function () {
-            this.inKey = 'start_stop';
+            this.inKey = 'reverse';
+            this.onlyOnPress = false;
```

Comment:
inValueScale() still returns the opposite/inverse of the current value, i.e. it toggles the value. As far as I can imagine this should have the same effect as setting it to 1 when pressed and to 0 when released. That's what I called a 'switch' button. The 'toggle' behaviour used to simulate a 'switch' here will get out of sync if 'reverse' ever contains an unexpected value or if one button event is lost somehow. Setting the value directly depending on the button event would be easier to understand and also more reliable.